chsdi/locale/fi/LC_MESSAGES/chsdi.po

```diff
@@ -8766,6 +8775,18 @@ msgstr "Standort"
 msgid "tt_ch.bfe.statistik-wasserkraftanlagen_name"
 msgstr "Name"
 
+msgid "tt_ch.bfe.statistik-wasserkraftanlagen_performancegeneratormaximum"
```

Comment:
please follow the convention.
ch.bfe.statistik-wasserkraftanlagen.performancegeneratormaximum

and you don't need to add the translations it is automatic when you fill up, ms_item.